test/test_utils.py

```diff
@@ -136,6 +137,93 @@ def test_vcap_credentials():
     assert authenticator is None
     del os.environ['VCAP_SERVICES']
 
+    vcap_services = '{"test":[{"name": "testname",\
+        "credentials":{ \
+        "url":"https://gateway.watsonplatform.net/compare-comply/api",\
+        "username":"bogus username", \
+        "password":"bogus password"}}]}'
+
+    os.environ['VCAP_SERVICES'] = vcap_services
+    authenticator = get_authenticator_from_environment('testname')
+    assert authenticator is not None
```

Comment:
In these situations, we should also verify that we got back the correct Authenticator type:
```
assert isinstance(authenticator, BasicAuthenticator)
```

Reply:
done